src/js/utils/spec-helpers/ajax-helpers.js

```diff
@@ -1,9 +1,20 @@
 define(['sinon', 'underscore', 'URI'], function(sinon, _, URI) {
     'use strict';
 
-    var fakeRequests, expectRequest, expectJsonRequest, expectPostRequest, expectRequestURL,
+    var XML_HTTP_READY_STATES, fakeServer, createFakeRequests, withFakeRequests, fakeRequests, currentRequest,
+        expectRequest, expectNoRequests, expectJsonRequest, expectPostRequest, expectRequestURL, skipResetRequest,
```

Comment:
It seems like theres's some redundancy between these variables and the exported module object at the bottom of the file. You could cut down on this long list of variable definitions and remove the need to keep it in sync with the module object by having two variables, `XHR_READY_STATES`, and `AjaxHelpers`. `AjaxHelpers` could then be the module object which is returned, and these functions could be defined as methods directly on that object.